### ISSUE TITLE
could not run pod install ,File exists 

### ISSUE THREAD

**Issue by agonmati** [CocoaPods/CocoaPods#5116]
### Report

* What did you do?

* What did you expect to happen?

* What happened instead?

### Stack

```
   CocoaPods : 0.39.0
        Ruby : ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin15]
    RubyGems : 2.6.2
        Host : Mac OS X 10.11.4 (15E65)
       Xcode : 7.3 (7D175)
         Git : git version 2.6.4 (Apple Git-63)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ dd8152e0a089fbf8f03a4c5206251a515051311a
```

### Plugins

```
cocoapods-plugins : 0.4.2
cocoapods-search  : 0.1.0
cocoapods-stats   : 0.6.2
cocoapods-trunk   : 0.6.4
cocoapods-try     : 0.5.1
```

### Podfile

```ruby
# Uncomment this line to define a global platform for your project
platform :ios, '8.0'
# Uncomment this line if you're using Swift
use_frameworks!

target 'ATTO' do

use_frameworks!
pod 'SQLite.swift', '~> 0.10.1'


source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '8.0'
use_frameworks!
pod 'Kingfisher', '~> 2.2'


platform :ios, '8.0'
use_frameworks!
target ‘ATTO’ do
    pod 'SwiftyJSON', :git => 'https://github.com/SwiftyJSON/SwiftyJSON.git'
end


pod 'Refresher'


pod 'Charts'


pod 'Gifu'
use_frameworks!


target ‘ATTO’ do
  pod 'AFImageHelper', '~> 3.0'
end


use_frameworks!
pod "DKChainableAnimationKit", "~> 1.6.0"


pod 'MGSwipeTableCell'


target ‘ATTO’ do
  pod 'Toucan', '~> 0.5'
end





end

target 'ATTOTests' do

end

```
<img width="585" alt="screen shot 2016-04-04 at 14 22 21" src="https://cloud.githubusercontent.com/assets/18262639/14247762/b604c780-fa70-11e5-9a9f-00ef60fb1bb5.png">

**Comment by orta** [comment]
It looks like you've just started using CocoaPods, so I think you need to read the guides about what a Podfile is : https://guides.cocoapods.org/using/getting-started.html and how it works. :+1:

**Comment by agonmati** [comment]
I have installed this podfile earlier, and I was using it during last 3 moths,I had a problem archiving app with some libraries after I updated my version of xcode to 7.3 but in simulator it worked fine and i tried to delete the podfile and reinstall again,when i tried installing it again it shows me this error.

**Comment by orta** [comment]
OK, sure, can you please include the stack trace for the error?

**Comment by agonmati** [comment]
### Error

```
Errno::EEXIST - File exists - /Users/agon/Desktop/ClockIn/Pods/Target Support Files/Pods-ATTO-ATTO
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer/target_installer.rb:86:in `mkdir'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer/target_installer.rb:86:in `mkdir'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer/target_installer.rb:86:in `create_support_files_dir'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb:14:in `block in install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:140:in `message'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer/target_installer/aggregate_target_installer.rb:12:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:621:in `block (2 levels) in install_libraries'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:618:in `each'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:618:in `block in install_libraries'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:140:in `message'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:611:in `install_libraries'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:156:in `block in generate_pods_project'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:153:in `generate_pods_project'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:111:in `install!'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:101:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/lib/cocoapods/command.rb:47:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.39.0/bin/pod:44:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'

**Comment by agonmati** [comment]
Sorry , it was just syntax error on podfile

**Comment by segiddins** [comment]
This has been fixed in the 1.0 betas.
